src/main/java/org/openrewrite/java/migrate/util/RelocateSuperCall.java

```diff
@@ -0,0 +1,68 @@
+package org.openrewrite.java.migrate.util;
+
+
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Preconditions;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.search.UsesJavaVersion;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.java.tree.Statement;
+
+import java.util.List;
+
+@EqualsAndHashCode(callSuper = false)
+@Value
+public class RelocateSuperCall extends Recipe {
```

Comment:
Might make sense to reference this immediately in the Java 25 upgrade recipe:
https://github.com/openrewrite/rewrite-migrate-java/blob/cb161e6fb27a5b5c1a5374aa750fe21baefa9f5f/src/main/resources/META-INF/rewrite/java-version-25.yml#L18-L32